sdk/src/wallet/operations/transaction/build_transaction.rs

```diff
@@ -36,8 +36,12 @@ where
 
         let mut inputs: Vec<Input> = Vec::new();
         let mut context_inputs = HashSet::new();
+        let mut mana_rewards = 0;
```

Comment:
What's the point of this?

Reply:
it accumulates the total mana rewards for semantic validation (which isn't part of this PR)

Reply:
added a comment